assets/js/blocks/breadcrumbs/block.json

```
@@ -0,0 +1,43 @@
+{
+	"name": "woocommerce/breadcrumbs",
+	"version": "1.0.0",
+	"title": "Store Breadcrumbs",
```

I think the latest decision on the block name was `WooCommerce Breadcrumbs` to explicitly differentiate these Breadcrumbs from any other Breadcrumbs blocks. 

```suggestion
	"title": "WooCommerce Breadcrumbs",
```

For reference: pdnLyh-2Y6-p2#comment-1942

---

Thanks! I was under the impression the decision was to stick with **Store**, based on the description on #8063. 

I see on that discussion that for the notices, for example, we decided to go with **Store** instead of **WooCommerce** as per Vivien it was adding unnecessary branding. Ref: pdnLyh-2Y6-p2#comment-1967

Paging @vivialice to confirm here!

---

> I see on that discussion that for the notices, for example, we decided to go with Store instead of WooCommerce as per Vivien it was adding unnecessary branding.

This should apply to Breadcrumbs as well. I'm sorry for the confusion.

---

Thanks for confirming! I'm keeping it as Store.